packages/ckeditor5-enter/src/enter.js

```diff
@@ -46,7 +46,8 @@ export default class Enter extends Plugin {
 			}
 
 			editor.execute( 'enter' );
+
 			view.scrollToTheSelection();
-		}, { priority: 'low' } );
```

Comment:
I'd keep this. We listened on `low` to make sure that if someone didn't pass any priority (and want to override this), that they don't have to do anything else.

Same in: shift enter, delete and perhaps somewhere else.